packages/sol-cov/README.md

```diff
@@ -8,6 +10,8 @@ yarn add -D @0xproject/sol-cov
 
 ## Usage
 
+Sol-cov uses transaction traces in order to figure out which lines of Solidity source code have been covered by your tests. In order for it to gather these traces, you must add the `CoverageSubprovider` to the [ProviderEngine](https://github.com/MetaMask/provider-engine) instance you use when running your Solidity tests. If you're unfamiliar with ProviderEngine, please read the [Web3 Provider explained](https://0xproject.com/wiki#Web3-Provider-Explained) wiki article.
+
 ```
 import { CoverageSubprovider } from '@0xproject/sol-cov'
 
```

Comment:
I think we should write a comment explaining what `defaultFromAddress` should be, and that it needs to be funded and "unlocked" for the `sendTransaction` calls we do.